src/main/java/org/interledger/setup/model/ReceiverQuery.java

```diff
@@ -0,0 +1,5 @@
+package org.interledger.setup.model;
+
+public interface ReceiverQuery {
+
```

Comment:
The idea here is to define SPSP as one of potentially many setup protocols. So a receiver does not necessarily have a URI (this is defined in SpspReceiver).

Different setup protocols will have different ways to return a Receiver so another SetupService might actually just build it locally.